nemesis/arena_battles.py

```diff
@@ -461,7 +462,7 @@ def battles_1():
     restock_downs()
     nemesis.arenaSelect.arena_menu_select(4)
     memory.main.full_party_format("kilikawoods1")
-    nemesis.menu.tidusSlayer(odPos=0)
+    menu.tidus_slayer(odPos=0)
```

Comment:
These change have not been tested. It'll take me two hours to get here again. I really ought to figure out how to get a save at the start of nem arena. So I can just jump right here.